pom.xml

```diff
@@ -230,8 +230,8 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <source>1.6</source>
-                    <target>1.6</target>
+                    <source>1.7</source>
+                    <target>1.7</target>
```

Comment:
Yes bumping to parent pom 2 would cause too much unrelated changes and make the review even harder/longer ;)